apps/src/feedback.js

```diff
@@ -534,6 +558,22 @@ FeedbackUtils.prototype.displayFeedback = function(
   }
 };
 
+function logDialogActions(event, options, isPerfect) {
+  if (experiments.isEnabled(experiments.FINISH_DIALOG_METRICS)) {
+    firehoseClient.putRecord({
+      study: FIREHOSE_STUDY,
+      study_group: firehose_event,
+      event: event,
```

Comment:
what's the difference between `event` and `firehose_event`?

Reply:
That's poor naming on my part - the firehose_event represents the type of dialog ("validate", "share", "share_validate", etc.) that the events are firing from. The event is whether the user clicked on the "replay" button or the 'next level' button. I'll update the firehose_event to be a more accurate description.